src/psyclone/tests/kernel_transformation_test.py

```diff
@@ -483,6 +483,37 @@ def test_no_inline_global_var():
     kernels = sched.walk(Kern)
     with pytest.raises(TransformationError) as err:
         _, _ = inline_trans.apply(kernels[0])
-    assert ("'kernel_with_global_code' contains accesses to data that are not "
-            "captured in the PSyIR Symbol Table (Undeclared reference 'alpha'"
-            " found" in str(err))
+    assert ("'kernel_with_global_code' contains accesses to data (variable "
+            "'alpha') that are not captured in the PSyIR Symbol Table(s) "
+            "within KernelSchedule scope." in str(err))
+
+# Class KernelTrans
+
+# Method validate
```

Comment:
If you really need these 2 comments (the test function already has this information in the name), move the 2 linebreaks separation before the comments instead of after them.

Reply:
Changed.